azure-pipelines.sds.pr-release.yml

```diff
@@ -34,39 +34,39 @@ stages:
-            parameters:
-              dotnetVersion: ${{ variables.dotnetVersion }}
-              nugetConfigPath: nuget.config
-              appName: ${{ variables.appName }}
-              dockerComposeTestFile: docker-compose.tests.yml
-              publishNodeTests: true
-              nodeTestResultFolder: $(System.DefaultWorkingDirectory)/VideoWeb/VideoWeb/ClientApp
-              sonarExtraProperties: |
-                sonar.exclusions=**/node_modules/**, **/*.spec.ts, *.spec.ts, **/ClientApp/src/*, **/ClientApp/coverage/**/**, **/ConfigureServicesExtensions.cs, **/Startup.cs, **/Program.cs, **/ClientApp/src/scripts/*.js, **/VideoWeb/VideoWeb.AcceptanceTests/Videos/**.y4m, **/ClientApp/src/scripts/**, **/ClientApp/src/app/testing/**, **/ClientApp/src/app/sass/**,
-                sonar.typescript.exclusions=**/node_modules/**, **/typings.d.ts, **/main.ts,**/environments/environment*.ts, **/*routing.module.ts, **/api-client.ts, **/app-insights-logger.service.ts
-                sonar.coverage.exclusions=**/Testing.Common/**, VideoWeb/Views/*, VideoWeb/Pages/*, VideoWeb.AcceptanceTests/*, **/VideoWeb.UnitTests/**, **/VideoWeb.Common/**, **/VideoWeb/Swagger/**, **/VideoWeb/Extensions/**, **/ClientApp/src/scripts/**, **/ClientApp/src/app/testing/**, **/ClientApp/src/app/vh-officer/helper.ts, **/ClientApp/src/app/services/events.service.ts, **/ClientApp/src/app/services/api/video-web.service.ts, **/ClientApp/src/app/waiting-space/analogue-clock/analogue-clock.component.ts, **/app-insights-logger.service.ts,**/video-filter.service.ts, **/video-filter.service.ts, **/audio-only-image.service.ts
-                sonar.cs.opencover.reportsPaths=$(System.DefaultWorkingDirectory)/coverage.opencover.xml
-                sonar.javascript.lcov.reportPaths=**/ClientApp/coverage/lcov.info
-                sonar.cpd.exclusions=**/tests/WRTestComponent.ts, **/joh-waiting-room/**, **/ClientApp/src/scripts/**
-                sonar.issue.ignore.multicriteria=e1, e2, e3, e4, e5, e6
-                sonar.issue.ignore.multicriteria.e1.ruleKey=typescript:S107
-                sonar.issue.ignore.multicriteria.e1.resourceKey=**/*.ts
-                sonar.issue.ignore.multicriteria.e2.ruleKey=typescript:S1874
-                sonar.issue.ignore.multicriteria.e2.resourceKey=**/*.ts
-                sonar.issue.ignore.multicriteria.e3.ruleKey=typescript:S6544
-                sonar.issue.ignore.multicriteria.e3.resourceKey=**/*.ts
-                sonar.issue.ignore.multicriteria.e4.ruleKey=csharpsquid:S107
-                sonar.issue.ignore.multicriteria.e4.resourceKey=**/*.cs
-                sonar.issue.ignore.multicriteria.e5.ruleKey=typescript:S6861
-                sonar.issue.ignore.multicriteria.e5.resourceKey=**/tests/**/*.ts
-                sonar.issue.ignore.multicriteria.e6.ruleKey=typescript:S2933
-                sonar.issue.ignore.multicriteria.e6.resourceKey=**/*.ts
+      # - job: UnitAndIntegrationTests
```

Comment:
Not running the tests for this build?

Reply:
I turned it off temporarily since the I needed to change URLs and the unit test stage took too long to move onto the stage I needed to actually test. I've restored them now